### DISCUSSION TITLE
RHF does not work with Fluent UI components

### DISCUSSION THREAD

**Discussion by AliSharafi** [react-hook-form/react-hook-form#3073] [Q&A]
Hello every one
I need your thoughts on this please. I am trying to make RHF work with [Fluent UI](https://github.com/microsoft/fluentui) components but I don't get TextField value on form submit. I have created this sand box to show case the issue 
https://codesandbox.io/s/wizardly-maxwell-m6swk?file=/src/index.tsx

As you can see here a normal input is fine but Fluent UI text field is not 
Thank you for helpful comments

**Comment by bluebill1049** [comment]
try with Controller, it probably doesn't expose `ref`
